huaweicloud/endpoints.go

```diff
@@ -38,65 +38,61 @@ var allServiceCatalog = map[string]ServiceCatalog{
 		Name:    "ecs",
 		Version: "v1",
 	},
-	"computeV11": ServiceCatalog{
+	"ecsv11": ServiceCatalog{
 		Name:    "ecs",
 		Version: "v1.1",
 	},
-	"computeV2": ServiceCatalog{
+	"ecsv21": ServiceCatalog{
 		Name:    "ecs",
 		Version: "v2.1",
 	},
-	"autoscalingV1": ServiceCatalog{
+	"autoscalingv1": ServiceCatalog{
 		Name:    "as",
 		Version: "autoscaling-api/v1",
 	},
-	"imageV2": ServiceCatalog{
+	"imagev2": ServiceCatalog{
 		Name:             "ims",
 		Version:          "v2",
 		WithOutProjectID: true,
 	},
-	"cceV3": ServiceCatalog{
+	"ccev3": ServiceCatalog{
 		Name:    "cce",
 		Version: "api/v3/projects",
 	},
-	"cceAddonV3": ServiceCatalog{
+	"cceaddonv3": ServiceCatalog{
 		Name:             "cce",
 		Version:          "api/v3",
 		WithOutProjectID: true,
 	},
-	"cciV1": ServiceCatalog{
+	"cciv1": ServiceCatalog{
 		Name:             "cci",
 		Version:          "apis/networking.cci.io/v1beta1",
 		WithOutProjectID: true,
 	},
-	"FgsV2": ServiceCatalog{
+	"fgsv2": ServiceCatalog{
 		Name:    "functiongraph",
 		Version: "v2",
 	},
 	// ******* client for Compute end  *******//
 
 	// ******* client for storage start ******//
-	"blockStorageV2": ServiceCatalog{
+	"blockStoragev2": ServiceCatalog{
 		Name:    "evs",
 		Version: "v2",
 	},
-	"blockStorageV3": ServiceCatalog{
+	"blockStoragev3": ServiceCatalog{
```

Comment:
the tag should be `blockstoragev2` and  `blockstoragev3`

Reply:
I will submit a new request to fix this later, this request will be closed.